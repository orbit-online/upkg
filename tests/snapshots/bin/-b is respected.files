[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  not-default-linked.sh -> ../.packages/scattered-executables.tar@5f8b45aba9f6113f79092a5dd28a4a345fed96155b3860d4608b98eb83b39888/bin/not-default-linked.sh
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  scattered-executables.tar@5f8b45aba9f6113f79092a5dd28a4a345fed96155b3860d4608b98eb83b39888
│   │       ├── [drwxr-xr-x]  bin
│   │       │   ├── [-rwxr-xr-x]  bin-exec.sh
│   │       │   └── [-rwxr-xr-x]  not-default-linked.sh
│   │       ├── [-rwxr-xr-x]  root-exec.sh
│   │       ├── [drwxr-xr-x]  tools
│   │       │   ├── [-rwxr-xr-x]  another-tool
│   │       │   ├── [-rw-r--r--]  not-an-exec
│   │       │   └── [-rwxr-xr-x]  tools-exec
│   │       └── [-rw-r--r--]  upkg.json
│   └── [lrwxrwxrwx]  scattered-executables.tar -> .packages/scattered-executables.tar@5f8b45aba9f6113f79092a5dd28a4a345fed96155b3860d4608b98eb83b39888
└── [-rw-r--r--]  upkg.json
