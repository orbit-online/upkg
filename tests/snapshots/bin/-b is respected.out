upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/default/scattered-executables.tar'
